main.go

```diff
@@ -20,29 +20,28 @@ package main
 
 import (
 	"flag"
-	"os"
-	"os/signal"
-	"syscall"
-	"time"
-
+	"fmt"
 	"github.com/golang/glog"
-
 	apiv1 "k8s.io/api/core/v1"
 	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	"k8s.io/apimachinery/pkg/util/clock"
 	clientset "k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
-
 	crdclientset "k8s.io/spark-on-k8s-operator/pkg/client/clientset/versioned"
 	crdinformers "k8s.io/spark-on-k8s-operator/pkg/client/informers/externalversions"
 	"k8s.io/spark-on-k8s-operator/pkg/controller/scheduledsparkapplication"
 	"k8s.io/spark-on-k8s-operator/pkg/controller/sparkapplication"
 	"k8s.io/spark-on-k8s-operator/pkg/crd"
 	ssacrd "k8s.io/spark-on-k8s-operator/pkg/crd/scheduledsparkapplication"
 	sacrd "k8s.io/spark-on-k8s-operator/pkg/crd/sparkapplication"
+	"k8s.io/spark-on-k8s-operator/pkg/util"
 	"k8s.io/spark-on-k8s-operator/pkg/webhook"
+	"os"
```

Comment:
Built-ion packages should come at the top.

Reply:
ran goimports on this and other files. I was using gofmt earlier which ended up reworking the imports in alphabetical order.